examples/private_ai_studio/main.tf

```diff
@@ -55,16 +137,32 @@ module "aihub" {
   }
 
   container_registry = {
-    create_new     = true
-    zone_redundant = false
+    create_new = false
   }
 
   key_vault = {
     create_new = true
+    private_endpoints = {
+      vault = {
+        name                          = "pe-keyvault-vault"
+        subnet_resource_id            = module.virtual_network.subnets["private_endpoints"].resource_id
+        private_dns_zone_resource_ids = [module.private_dns_keyvault_vault.resource_id]
+        inherit_lock                  = false
+      }
+    }
   }
 
   storage_account = {
     create_new = true
+    private_endpoints = {
+      blob = {
```

Comment:
Shall we create a private endpoint for file as well? (see: https://learn.microsoft.com/en-us/azure/ai-studio/media/how-to/network/azure-ai-network-outbound.png)